Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util/factory.go

```diff
@@ -209,6 +209,9 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 				}
 				return &clientSwaggerSchema{client, api.Scheme}, nil
 			}
+			if filenameFlag := flags.Lookup("filename"); filenameFlag != nil && filenameFlag.Changed {
```

Comment:
This forces filename validation on all objects processed (users can pass filename and something else).  Are there any scenarios in which this would break a command that uses resource builder multiple times in a row?